integration_tests/pages/refer/confirmPerson.ts

```diff
@@ -0,0 +1,31 @@
+import { personUtils } from '../../../server/utils'
+import Page from '../page'
+import type { Person } from '@accredited-programmes/models'
+
+export default class ConfirmPersonPage extends Page {
+  person: Person
+
+  constructor(args: { person: Person }) {
```

Comment:
Where we only have one argument in the course page, we aren't using `args`, but perhaps we should for consistency

Reply:
Have tweaked this to be a single arg - we can expand if the page needs to change in future.